R/reactives.R

```diff
@@ -1441,6 +1440,29 @@ setAutoflush <- local({
   }
 })
 
+
+#' Activate reactivity in the console
+#'
+#' This is an experimental feature that allows you to enable reactivity
+#' at the console, for the purposes of experimentation and learning.
+#'
+#' @keywords internal
+#' @param enabled Turn console reactivity on or off?
+#' @export
+#' @examples
+#' reactiveConsole(TRUE)
+#' x <- reactiveVal(10)
+#' y <- observe({
+#'   message("The value of x is ", x())
+#' })
+#' x(20)
+#' x(30)
+#' reactiveConsole(FALSE)
+reactiveConsole <- function(enabled) {
```

Comment:
Time based reactives (i.e. `invalidateLater()`) won't currently work--but they could...

Reply:
`timerCallbacks$executeElapsed()` doesn't handle time based reactives?

Reply:
Well it does, but, it's only invoked right at the moment the task callback is called. We could use `later` to invoke it "spontaneously".

Reply:
Ah, yeah, makes sense. That doesn't block this PR though, right?

Reply:
Not at all.